chromium_src/chrome/browser/ui/views/translate/translate_bubble_controller.cc

```diff
@@ -0,0 +1,27 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "chrome/browser/ui/views/translate/translate_bubble_controller.h"
+#include "brave/browser/ui/views/translate/brave_translate_bubble_view.h"
+#include "brave/components/translate/core/common/brave_translate_features.h"
+#include "brave/components/translate/core/common/buildflags.h"
+
+// static
```

Comment:
Can we make this static method be a free function? ctors of `BraveTranslateBubbleView` & `TranslateBubbleView` are public.

Non blocking note, this can be changed after merging this PR.

Reply:
Done in [a5e9221](https://github.com/brave/brave-core/pull/13498/commits/a5e92217fb8ea5c511c7a49ee7f290da77e08656)